### PR TITLE
Add tests for Attendance

### PR DESCRIPTION
**Pull request by vanwagonet**
refs: MBL-13251
affects: Teacher
release note: none

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1010342&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Teacher

[MBL-13251](https://instructure.atlassian.net/browse/MBL-13251)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.29% | 93.29% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: rn/Teacher/ios/TeacherTests/Attendance/AttendanceViewControllerTests.swift [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+        navigation = UINavigationController(rootViewController: controller)
+
+        api.mock(GetCustomColorsRequest(), value: APICustomColors(custom_colors: [
+            context.canvasContextID: "#008EE2", // electric
+        ]))
+        api.mock(GetCourseRequest(courseID: context.id), value: .make())
+        api.mock(GetCourseSectionsRequest(courseID: context.id, perPage: 100), value: [
+            .make(),
+            .make(name: "section 2"),
+        ])
```
Comment (armstrongnate) [inline rn/Teacher/ios/TeacherTests/Attendance/AttendanceViewControllerTests.swift]: For future reference, you can pass in a `Store` to `api.mock` so you don't have to match the request exactly. Eg:

```swift
api.mock(controller.colors, value: ...)
api.mock(controller.course, value: .make())
api.mock(controller.sections, value: [...])
```

Comment (vanwagonet) [reply rn/Teacher/ios/TeacherTests/Attendance/AttendanceViewControllerTests.swift]: That's nice. Looks like I need to fix a swiftlint error, so I may as well change these too.

**Pull request merged**